### ISSUE TITLE
[bug] PST Spell effects that should be temporary are permanent

### ISSUE THREAD

**Issue by chilvence** [gemrb/gemrb#641]
The bonus effects of many spells never expire, eg power of one can be used to permanently increase your strength 

Spells affected:

SPWI111 Shield
SPWI119 Submerge the Will
SPWI203 Black barbed shield
SPWI211 Strength
SPWI219 Power of One
SPWI312 Infernal Shield
SPWI406 Improved Strength
SPWI408 Shroud of Shadows
SPPR106 Protection from evil
SPPR302 Flamewalk

Some of these are reliant on unhardcoded projectiles. Could it be a simple oversight in how they were set up? I am not totally sure how to go about inspecting / editing them. I can probably figure it out, but I thought I would just as well make a bug report in case it turns out the solution is obvious to anyone.

**Comment by lynxlynxlynx** [comment]
Thanks, this sounds more like extra problems with timing modes, just like #637. Durations are set in effects, the projectiles are there only for delivery. 

Eg. shield has a delayed/permanent mode, causing both a lag before application and the permanence. The description says 25s (5 rounds) / level and I guess it's similar for these others on the list (ok, Submerge the Will has a bug in its description for sure, but still). But then Black barbed shield, Strenght, Power of One have no useful info, so perhaps PST did encode some of it in the projectile.

**Comment by chilvence** [comment]
I did wonder, but according to the IESDP the PRO format only started with BG2, so who knows what sort of behind the scenes stuff was handled by the game engine. A recurring memory keeps nagging me, I am sure I read an interview once that said the magic stuff was essentially unfinished when BI first got the game.

I did notice that some of the timings are altered again in PSTOpcodes.cpp, which will probably add to the confusion.

I have a vague theory that one timing mode could mean seconds and the other could mean tics, but I haven't tested it with the original game yet.
